app/mappers/cocina/work_events_mapper.rb

```diff
@@ -29,7 +29,7 @@ def creation_date_event_params
       date = if create_date_type == 'single'
                create_date_single.to_s
              else
-               "#{create_date_range_from}/#{create_date_range_to}"
+               [create_date_range_from, create_date_range_to].filter_map(&:to_s).join('/').presence
```

Comment:
Trivial (non-blocking) quibble/suggestion:

```suggestion
               [create_date_range_from, create_date_range_to].filter_map(&:presence).join('/').presence
```

Reply:
I'm not convinced that change would work as intended. The forms will never be `nil`; they just might have `nil` attributes that when `to_s` is called might be `nil`. (There is logic in the forms about how to create a string.)

Reply:
Since the quibble is minor and given ^^, I'm going to leave as is.